### DISCUSSION TITLE
Adding events

### DISCUSSION THREAD

**Discussion by SilverG88** [Keriew/augustus#1139] [General]
Hello all, I am fairly new and was wondering how can we add more Emperor Requests? The max is 20. Would be nice in we could add these through .XML

Thank you. :)

**Comment by crudelios** [comment]
Hi,

I've been slowly changing the code to support an unlimited number of requests right from the map editor, but progress has been slow due to lack of time.

Anyway, that's the next planned feature, so it shouldn't take too long hopefully.

**Reply by SilverG88** [reply]
Is there a guide on how to make events in XML?

**Reply by crudelios** [reply]
You can see xml examples here: https://github.com/Keriew/augustus/tree/master/res/editor_examples

Check for the xml files with example events.
